Algorithmia/handler.py

```diff
@@ -0,0 +1,98 @@
+import sys
+import json
+import base64
+import traceback
+import six
+
+
+class Handler(object):
+
+    def __init__(self, apply_func, load_func=None):
+        """
+        Creates the handler object
```

Comment:
This documentation is great. Is there a way to assert on the arity of the functions in Python, and return a descriptive error message, such that we could fail fast if I attempt to construct a `Handler` with a `load_func` that takes an argument.